lib/oli_web/controllers/user_authorization_controller.ex

```diff
@@ -23,6 +23,20 @@ defmodule OliWeb.UserAuthorizationController do
   def new(conn, %{"provider" => provider}) do
     config = conn.assigns.assent_auth_config
 
+    conn =
```

Comment:
Would it be better to use a user's platform roles or `can_create_sections` field to determine where to redirect to?

Reply:
The first time an "instructor" signs up via social login, the `can_create_sections ` field is not automatically set. So, instructor signup will always initially land the user `/workspaces/student` until an admin sets that field. An option would be automatically setting the field, which is not wise.

Reply:
Ah, makes sense